### PR TITLE
Add PSS toggle switch

### PR DESCRIPTION
**Pull request by vxav**
This PR:

- adds/changes/removes etc

### Testing

Description on how {APP-NAME} can be tested.

- [x] fresh install works
- [x] upgrade from previous version works

#### Other testing

Description of features to additionally test for {APP-NAME} installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] Update `/examples` if required.

### CODE REVIEW

**Review by glitchcrab — APPROVED** [review]
seems to make sense to me

File: helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml [hunk 1]
```diff
@@ -24,10 +24,10 @@ spec:
       chart: cloud-provider-cloud-director
       # used by renovate
       # repo: giantswarm/cloud-provider-cloud-director-app
-      version: 0.2.9
+      version: 0.2.9-d0437b6bea8531e199cab35e29cf9f751d48815d
```
Comment (erkanerol) [inline helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml:27]: Please let me know if everything is fine regarding https://github.com/giantswarm/cloud-provider-cloud-director-app/pull/78#issuecomment-1905720470 then I can approve without worrying about it :D

Comment (vxav) [reply helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml:27]: I want to merge this to `main` first in a separate PR.
https://github.com/giantswarm/cluster-cloud-director/pull/248

**Comment by vxav** [comment]
@erkanerol this is ready for review.
test cluster available in `gerbil/test1-25`

**Review by erkanerol — APPROVED** [review]

**Pull request merged**